assets/js/components/Dashboard/PaMessaging/NewPaMessagePage/NewPaMessagePage.tsx

```diff
@@ -8,7 +17,23 @@ import { useNavigate } from "react-router-dom";
 import moment from "moment";
 import DatePicker from "../../DatePicker";
 import TimePicker from "../../TimePicker";
-import { ArrowRightShort, PlusLg, VolumeUpFill } from "react-bootstrap-icons";
+import {
+  ArrowRightShort,
+  CheckCircleFill,
+  ExclamationTriangleFill,
+  PlusLg,
+  VolumeUpFill,
+} from "react-bootstrap-icons";
+import cx from "classnames";
+
+const MAX_TEXT_LENGTH = 2000;
+
+enum AudioState {
+  HasNotBeenPlayed,
+  IsPlaying,
+  HasBeenReviewed,
+  NeedsReview,
```

Comment:
What is the difference between `HasNotBeenPlayed` and `NeedsReview`? Playing the audio _is_ reviewing it, right? I'm a bit confused by the naming here since half the states refer to "playing" and half to "reviewing".

Reply:
`HasNotBeenPlayed` is used as the default state. Without some kind of default, the icon next to `Review audio` will change prematurely to a warning icon when the text in the boxes is updated before audio is played. Another idea I had was just using `null` as the default considering it is only used on page load. Didn't see it before, but I agree that the two names overlap a bit too much.

Reply:
I see. What do you think of this naming, then? (same order, just took a stab at different names)

```ts
enum AudioState {
  Unreviewed,
  Playing,
  Reviewed,
  Outdated,
}
```